packages/vue/src/utilities/directives/focus-trap-directive.js

```diff
@@ -0,0 +1,20 @@
+import { getFocusableChildrens, isFocusable, trap } from "./focus-trap/focus-trap.js";
+
+export const focusTrap = {
+  bind(el) {
+    const focusableChildrenElements = getFocusableChildrens(el);
+    el._keyHandler = function (e) {
+      if (e.key === "Tab") {
+        if (!isFocusable(e, focusableChildrenElements)) {
```

Comment:
After applying changes from https://github.com/DivanteLtd/storefront-ui/pull/1088/files#diff-45d744fac3449b0aabab678c3daf250dR17 you can check the length of `focusableChildrenElements` only, instead of using `isFocusable()`